### PR TITLE
Scan project cronjobs

### PR DESCRIPTION
**Pull request by joverlee521**

### CODE REVIEW

**Comment by joverlee521** [comment]
[Add DET cronjobs for SCAN projects in production](https://trello.com/c/EVdinOCu/413-add-det-cronjobs-for-scan-projects-in-production)

File: crontabs/id3c-production [hunk 1]
```diff
@@ -62,19 +62,19 @@ ENVD=/opt/backoffice/id3c-production/env.d
...
 # conditions) once locking is done by ID3C directly.
 #
-# Transform REDCap DETs to FHIR documents every 10 minutes, interleaving asymptomatic
-# and swab-n-send on the 5s (since they share a cache file).
-# Only process uw-retros on the hour, with the expectation that it should only run
-# once a day. Still schedule for every hour to capture any manual pulls that we may do.
-# Only process the old swab-n-send project at 5 past and process from new project the rest of time.
-# We can eventually delete the old project once it has been completely stopped.
-# Only process kiosks at 10 past because they are closed for now.
+# Transform REDCap DETs to FHIR documents every 10 minutes.
```
Comment (tsibley) [inline crontabs/id3c-production]: This comment isn't accurate anymore.

File: crontabs/id3c-production [hunk 2]
```diff
@@ -62,19 +62,19 @@ ENVD=/opt/backoffice/id3c-production/env.d
...
+# Ingest old SFS projects every hour, interleaving the projects so each get
+# ingested once every 6 hours. Keeping these in case any data corrections are made.
+0 0-18/6 * * * ubuntu flock -F /home/ubuntu/cache.pickle envdir $ENVD/deidentification envdir $ENVD/smartystreets envdir $ENVD/redcap-uw-retrospectives pipenv run id3c etl redcap-det uw-retrospectives --commit
```
Comment (tsibley) [inline crontabs/id3c-production]: The project used for UW retros will still be in use, so the "old" implication doesn't apply, but every 6h is fine I think.

**Comment by tsibley** [comment]
Deployed.

**Pull request merged**